docs/workflow.rst

```diff
@@ -3,255 +3,182 @@
+same script can be used. If the output of one of the previous commands was 'map_name.hd5', then use
 
   .. code-block:: console
 
-     $ ln -s /location/beast/beast/ beast
-
-
+     $ ./beast/tools/create_background_density_map.py tileplot map_name.hd5 -image image.fits --colorbar 'background'
 
-Datamodel_template.py
-=====================
 
-Before running the BEAST, you will need to modify this file to specify
-the required parameters for generating models and fitting data.
-These parameters are described in the beast :ref:`setup documentation
-<beast_setup_datamodel>`.  The fields for project, obsfile, astfile,
-filters, and basefilters will be filled in by beast_production_wrapper.py.
+*************
+Physics model
+*************
 
-***************************
-Beast_production_wrapper.py
-***************************
+Generate the full physics model grid.  This is needed for both the fitting and
+for generating the artificial star test (AST) inputs.  Note that you may want to
+use a coarser model grid for the ASTs.
 
-This is a wrapper for each of the commands described below.  You may
-choose to run each of those commands individually, but this
-conveniently packages them into one file.  If you use this wrapper, you
-should edit several items in the file:
-
-  * field_names: used to identify photometry files and create BEAST files
-  * gst_filter_names: labels for the filters used in your photometry
-    file (e.g., 'X_RATE')
-  * beast_filter_names: the corresponding long names used by the BEAST
-  * settings for the source density map: pixel size, filter, magnitude
-    range
-  * settings for the background map: pixel dimensions, reference image
-  * settings for splitting the catalog by source density: filter,
-    number of sources per file
-  * settings for the trimming/fitting batch scripts: number of files, nice level
+If you're creating a model grid that's so large it may not read into memory, you
+can use subgrids, which splits the grid into more manageable pieces.
 
-You can (and should!) read about the individual functions below before
-running beast_production_wrapper:
+To create a physics model grid with 5 subgrids:
 
   .. code-block:: console
 
-     $ run beast_production_wrapper.py
+     $ beast.tools.run.create_physicsmodel --nsubs=5
 
-The first thing it does is use datamodel_template.py to create a
-datamodel.py file.  This will be imported as needed in the functions
-called by the wrapper.  As noted above, five of the datamodel fields
-will be updated, so ensure that the other fields in
-datamodel_template.py have the desired values.
 
-The wrapper will proceed through each of the functions below.  At
-three points, you will need to manually run things independently of
-the wrapper.  It will not continue running subsequent functions until
-it finds that the necessary steps have been taken.
+*********************
+Artificial Star Tests
+*********************
```

Comment:
It would be good to link to the generating_asts.rst docs somewhere in this section.  This doc has more info (but may need updating?).

Reply:
Can do.  And unsurprisingly, that doc does need updating!